server/routing/routing.go

```diff
@@ -69,7 +82,8 @@ func (r *route) List(ctx context.Context, req *routingv1.ListRequest) (<-chan *r
 	// Use remote routing when:
```

Comment:
List method is not implemented properly at the moment. We should do this separately since the List needs to:
a) Query each requested label/query individually and wait via channel for responses
b) Construct a response object with `peerLabels = map[string][]string` that indicates `peer -> available labels`
c) Filter the response object where each item `len(peerLabels[peer]) > minLabelMatch`
d) Return the data via channel

Reply:
You're right. List will be refactored in separate PR.